app/_components/GuessDistanceModal.tsx

```diff
@@ -111,8 +112,8 @@ const GuessDistanceModal = ({
           {distanceToPin < thresholdDistance ? (
             <div className="p-4">
               <p className="my-2">
-                Nice Guessing! How about leaving a hint for someone else? Be
-                sure to be helpful! But don&apos;t just give it away!
+                {`Nice Guessing! How about leaving a hint for someone else?`} 
+                {`(Be sure to be helpful! But don't just give it away!)`}
```

Comment:
`don&apos;t` is for when you use text inside the HTML tag directly. it will turn into `don't` by default. but the way you fix this is ok too. but no need for double back ticks, you can just wrap it inside one backtick.
```suggestion
                {`Nice Guessing! How about leaving a hint for someone else? 
                (Be sure to be helpful! But don't just give it away!)`}
```